<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineSetupDialog</class>
 <widget class="QDialog" name="MachineSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MachineSetupDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>SN</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMachine">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>55</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Machine</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leSn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbMachine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>55</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbLanguage">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Language</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbLanguage">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbForPets">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>For Pets</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTestSpeed">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>145</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Speed</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTestSpeed">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>145</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDiluent">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbDiluent">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Diluent</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbLyse">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>235</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbLyse">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>235</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Lyse</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DigitLineEdit</class>
   <extends>LineEdit</extends>
   <header>digitlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
